### ISSUE TITLE
[Docs] Documentation for depending on published packages.

### ISSUE THREAD

**Issue by amnn** [MystenLabs/sui#5937]
There is currently no documentation for how to depend on an on-chain package, although we did supply [this answer](https://stackoverflow.com/questions/73958046/local-dependency-cannot-be-resolved) on Stack Overflow.  We should find a home for it in our docs as well.

**Comment by stefan-mysten** [comment]
@amnn / @ronny-mysten I assume this still needs to be done somewhere.

**Comment by amnn** [comment]
Yes, I don't think we have this topic addressed directly anywhere.
